### PR TITLE
Made musket wieldable.

### LINKED ISSUE #29577: Muskets lack wieldability
## Description
Despite being a longarm weapon, muskets cannot be wielded

**Reproduction**
Step 1. Spawn a musket
Step 2. Attempt to wield
Step 3. Eject bullet

**Screenshots**

**Additional context**
Might take a while because it needs a wield sprite

### PR DESCRIPTION
**Pull request by jonathanargo**
## About the PR

Made muskets wieldable.

## Why / Balance

Fixes #29577. Muskets, despite being a long arm weapon, were not wieldable like the other long arm weapons.

## Technical details

Simply added BaseGunWieldable to the musket, but this also required adding wielding sprites. Copied and reskinned the bolt gun wielding sprites to resemble the musket. Seems to be serviceable enough.

Note that there appears to be an existing bug affecting all (most?) of the wield sprites where they are not flipped correctly when facing the opposite direction from the hand the weapon is held in (e.g. held in right hand, mob facing left). This bug is reflected in these new sprites as well.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

musket_wield
 

## Breaking changes

N/A
**Changelog**

:cl:
- tweak: Muskets are now wieldable.

### CODE REVIEW

File: Resources/Textures/Objects/Weapons/Guns/Snipers/musket.rsi/meta.json [hunk 1]
```diff
@@ -15,6 +15,14 @@
       "name": "inhand-left",
       "directions": 4
     },
+    {
```
Comment (TheShuEd) [inline Resources/Textures/Objects/Weapons/Guns/Snipers/musket.rsi/meta.json:18]: pls update copyright

**Review by TheShuEd — APPROVED** [review]

File: Resources/Textures/Objects/Weapons/Guns/Snipers/musket.rsi/meta.json (hunk 1 above)
Comment (jonathanargo) [reply Resources/Textures/Objects/Weapons/Guns/Snipers/musket.rsi/meta.json:18]: That OK?

File: Resources/Textures/Objects/Weapons/Guns/Snipers/musket.rsi/meta.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Made by Pato Grone for SS14",
+  "copyright": "Made by Pato Grone and Jonathan Argo for SS14",
```
Comment (TheShuEd) [inline Resources/Textures/Objects/Weapons/Guns/Snipers/musket.rsi/meta.json:4]: Made by Pato Grone for SS14, Wielded states by Jonathan Argo

Comment (jonathanargo) [reply Resources/Textures/Objects/Weapons/Guns/Snipers/musket.rsi/meta.json]: Ah, of course, good idea, thanks!

**Pull request merged**